spec/functional/resource/registry_spec.rb

```diff
@@ -270,20 +309,55 @@ def create_deletable_keys
       end
     end
 
+    it "prepares the reporting data for action :create with only changed values" do
```

Comment:
Just write straight to the registry to set up the scenario and avoid false positives/negatives of running setup through the `registry_key` resource.